net.warsow.Warsow.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: net.warsow.Warsow
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: warsow
+rename-icon: warsow
+rename-desktop-file: warsow.desktop
+rename-appdata-file: warsow.appdata.xml
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
```

Comment:
If it uses SDL2, shouldn't it work fine with Wayland too?

Reply:
I don't know. Every machine I own will crash when logging into a Wayland session.